idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/screen/mode/iconpack/destination/IconPackDestinationViewModel.kt

```diff
@@ -27,6 +29,7 @@ class IconPackDestinationViewModel(
             copy(
                 iconPackDestination = path.absolutePathString(),
                 nextButtonEnabled = true,
+                predictedPackage = PackageExtractor.getFrom(path.absolutePathString()).orEmpty(),
```

Comment:
```suggestion
                predictedPackage = PackageExtractor.getFrom(path.invariantSeparatorsPathString).orEmpty(),
```

Reply:
make sense now

Reply:
I decided to add `invariantSeparatorsPathString` logic only into `PackageExtractor`